static/css/devhub/base.less

```diff
@@ -100,16 +164,17 @@ section {
       margin-left: 10px;
     }
   }
-}
 
-.Logo--secondary::before {
-  background: url('../../img/developers/addons-logo-black.svg') no-repeat left center;
-  background-size: contain;
+  .scheme-medium & {
+    background: url('../../img/developers/addons-logo-black.svg') no-repeat left center;
+    background-size: contain;
+  }
 }
 
 .DevHub-Navigation {
-  background-color: @primary-section-background;
-  padding: 0 20px 20px;
+  > div {
```

Comment:
Not necessarily but I feel like it made various things easier for me regarding to paddings and positioning things in combination of background colors.